crm_lead_project/wizard/crm_lead_to_opportunity.py

```diff
@@ -45,7 +42,26 @@ def action_apply(self):
         values.update({'lead_ids': leads.ids, 'user_ids': [self.user_id.id]})
         self._convert_opportunity(values)
         # Create the project
-        project_values = self._prepare_project_values(leads)
-        project = self.env['project.project'].create(project_values)
-        leads.write({'project_id': project.id})
+        for lead in leads:
+            project_values = self._prepare_project_values(leads)
+            if self.env.user.has_group(
```

Comment:
@grindtildeath pls, check for permission instead otherwise even if you have rights you cannot add a project w/ your user.
NOTE: generally speaking checking for groups instead of permissions is not a good pattern.